fe/fe-core/src/main/java/com/starrocks/catalog/IcebergTable.java

```diff
@@ -154,11 +159,22 @@ private void validate(Map<String, String> properties) throws DdlException {
         IcebergCatalogType type = icebergResource.getCatalogType();
         icebergProperties.put(ICEBERG_CATALOG, type.name());
         LOG.info("Iceberg table type is " + type.name());
+
+        String fileIOCacheMaxTotalBytes = copiedProps.get(IcebergCachingFileIO.FILEIO_CACHE_MAX_TOTAL_BYTES);
+        if (!Strings.isNullOrEmpty(fileIOCacheMaxTotalBytes)) {
+            icebergProperties.put(IcebergCachingFileIO.FILEIO_CACHE_MAX_TOTAL_BYTES, fileIOCacheMaxTotalBytes);
+            copiedProps.remove(IcebergCachingFileIO.FILEIO_CACHE_MAX_TOTAL_BYTES);
+        }
+
+        if (!copiedProps.isEmpty()) {
```

Comment:
why do we need to throw exception on unknown properties?

Reply:
I have change ,  I don't see that there is still operation in it  CUSTOM_CATALOG case.